pytest.ini

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts= --durations=50 --cov-config=.coveragerc --cov-report xml --cov=aiida
+addopts = --durations=50 --cov-config=.coveragerc --cov-report xml --cov=aiida
```

Comment:
While we are here (I know this wasn't added by this PR, but it was added recently) can we have these options just enabled on github? These are now also run everytime I run the tests locally and it takes a lot of extra time and outputs a lot of noise. This really disturbs me when developing locally, so I have to remove these every time. @ltalirz I think you added these. Can we just move these to the CI configuration?

Reply:
I agree that this would be better to _not_ have locally. I can look into changing this back.

Reply:
>  it takes a lot of extra time

I was not aware of that... is it the coverage check?
In that case I agree that the two `--cov` options can probably be moved to github...

> outputs a lot of noise

What it should be outputting in addition is test timings.
So far, nobody has been paying attention to them (we have "unit tests" that run several seconds), and so I think it is a good idea to show them when you run tests locally.

Reply:
> > outputs a lot of noise
> 
> What it should be outputting in addition is test timings.
> So far, nobody has been paying attention to them (we have "unit tests" that run several seconds), and so I think it is a good idea to show them when you run tests locally.

It would be better to be able to toggle this locally. It _is_ a lot of unneccesary noise if you're just trying to see why a test is failing. Or even better would be to filter this and only show timings above a certain threshold.

Reply:
We don't need to see the timings all the time though. Of course when there are new tests, it would be good to see that they are not super slow. But that is why we have the timings run on the CI so they can be checked before merging.

Reply:
> Or even better would be to filter this and only show timings above a certain threshold.

that would be ok if there currently aren't any tests that surpass it, or at the very least are then also addressed immediately to reduce their time, otherwise we would still be spammed every time with the same information

Reply:
> Or even better would be to filter this and only show timings above a certain threshold.

This is already the case. It shows the 50 slowest tests.

> It is a lot of unneccesary noise if you're just trying to see why a test is failing.

Ok, perhaps I'm using local tests differently - I usually just run a handful of them.

> that is why we have the timings run on the CI so they can be checked

They *can* be, but I guess we agree that what matter is whether they actually *are*.

Perhaps that is better done by sensibilizing developers on this topic, though. I didn't want the timings to be annoying, of course.

Reply:
> Or even better would be to filter this and only show timings above a certain threshold.

Ok, I guess you meant a cutoff in ms, which would be even better.
Unfortunately, I don't think this is currently supported by pytest https://docs.pytest.org/en/latest/usage.html#profiling-test-execution-duration

Reply:
50 is a bit of a high number as well, I think. At least for the local test runs. Just 5 would probably be better. But if it's a contested issue, I think we should decide on this within the developer team and not deal with it in this PR (or deal with it after a quick decision)?

Reply:
Feel free to adapt to your preferences. I agree it is very important not to annoy people :-)
You could set it to 5 for local testing and 50 on github actions; this will already have the effect of people getting used to looking at timings.

Feel free to proceed without another review from my side. Cheers!